server/env.ts

```diff
@@ -385,6 +385,31 @@ export class Environment {
     environment.GOOGLE_ANALYTICS_ID
   );
 
+  /**
+   * A Umami analytics domain to track user interactions.
+   */
+  @Public
+  @IsOptional()
+  public UMAMI_SCRIPT_DOMAIN = this.toOptionalString(
```

Comment:
There is no need for any of these additions in env, as it can all be configured through the integration settings page – the existing `GOOGLE_ANALYTICS_ID` here is just for backwards compatibility to much older versions of Outline

Reply:
Ok, I can choose to remove it if I don't need it, I had hoped that, when the ops installed it, it would be done with a one-click deployment, eliminating the need for an administrator to log in and configure the integration.